packages/web-app-files/tests/unit/helpers/route.spec.js

```diff
@@ -1,48 +1,100 @@
+    })
   })
 
   describe('isSharedWithMeRoute', () => {
-    it.todo('should return "true" if given route name is "files-shared-with-me"')
-    it.todo('should return "false" if given route name is not "files-shared-with-me"')
+    it('should return "true" if given route name is "files-shared-with-me"', () => {
+      expect(isSharedWithMeRoute({ name: 'files-shared-with-me' })).toBe(true)
+    })
+    it('should return "false" if given route name is not "files-shared-with-me"', () => {
+      expect(isSharedWithMeRoute({ name: 'files-personal' })).toBe(false)
+    })
   })
 
   describe('isSharedWithOthersRoute', () => {
-    it.todo('should return "true" if given route name is "files-shared-with-others"')
-    it.todo('should return "false" if given route name is not "files-shared-with-others"')
+    it('should return "true" if given route name is "files-shared-with-others"', () => {
+      expect(isSharedWithOthersRoute({ name: 'files-shared-with-others' })).toBe(true)
+    })
+    it('should return "false" if given route name is not "files-shared-with-others"', () => {
+      expect(isSharedWithOthersRoute({ name: 'files-shared-with-me' })).toBe(false)
+    })
   })
 
   describe('isAnySharedWithRoute', () => {
-    it.todo('should return "false" if given route is not "SharedWithMe" and "SharedWithOthers"')
-    it.todo('should return "true" if given route is "SharedWithMe"')
-    it.todo('should return "true" if given route is "SharedWithOthers"')
+    it('should return "false" if given route is not "SharedWithMe" and "SharedWithOthers"', () => {
+      expect(isAnySharedWithRoute({ name: 'files-personal' })).toBe(false)
+    })
+    it('should return "true" if given route is "SharedWithMe"', () => {
+      expect(isAnySharedWithRoute({ name: 'files-shared-with-me' })).toBe(true)
+    })
+    it('should return "true" if given route is "SharedWithOthers"', () => {
+      expect(isAnySharedWithRoute({ name: 'files-shared-with-others' })).toBe(true)
+    })
   })
 
   describe('isPublicFilesRoute', () => {
-    it.todo('should return "true" if given route name is "files-public-list"')
-    it.todo('should return "false" if given route name is not "files-public-list"')
+    it('should return "true" if given route name is "files-public-list"', () => {
+      expect(isPublicFilesRoute({ name: 'files-public-list' })).toBe(true)
+    })
+    it('should return "false" if given route name is not "files-public-list"', () => {
+      expect(isPublicFilesRoute({ name: 'files-shared-with-others' })).toBe(false)
+    })
   })
 
   describe('isPublicPage', () => {
-    it.todo('should return "false" if given route has no meta information')
-    it.todo('should return "false" if given route meta auth is "true"')
-    it.todo('should return "true" if given route meta auth is "false"')
+    it('should return "false" if given route has no meta information', () => {
+      expect(isPublicPage({ name: 'files-public-list' })).toBe(false)
```

Comment:
files-public-list returns `false` when checked if it is a public page? 🤔 we need to check our code, it's not the test that is falsy I think